docs/admin_api/user_admin_api.rst

```diff
@@ -125,10 +127,10 @@ This API returns information about a specific user account.
 
 The api is::
 
-    GET /_synapse/admin/v1/whois/<user_id> (deprecated)
     GET /_synapse/admin/v2/users/<user_id>
```

Comment:
I am not sure if this is a mistake in the documentation.
`GET /_synapse/admin/v1/whois/<user_id>` lists informations about the connections and `GET /_synapse/admin/v2/users/<user_id>`  about the user. This are different APIs.